### ISSUE TITLE
Can I use PDB70 instead of PDB100?

### ISSUE THREAD

**Issue by SudeshiS** [sokrypton/ColabFold#492]
## Current Behavior
Hi,
I have been running ColabFold v1.5.2-patch to predict a protein complex. I have noticed that the predicted structure I was getting with the previous PDB70 database is much better than the one I am getting with the current PDB100 database. Is there a way to run this prediction with PDB70 now? Or could this change be due to some parameter change? I have been using the default settings. I have already tried both complete and greedy pairing.

## Context
I have some new analyses to do and I would like to do it in the same way as I did before ( before the database got updated to pdb100) which gave me a better prediction.

## Your Environment
Include as many relevant details about the environment you experienced the bug in.
* Git commit used
* If you run it on a local system. Please add the server specifications
* Operating system and version:

**Comment by cdelam** [comment]
I agree with you,. I have the same problem to finish some analyses. But I think other parameters have been changed. Have a look  into the file attached to see differences on the same job.
[Results.pdf](https://github.com/sokrypton/ColabFold/files/12440856/Results.pdf)

**Comment by sokrypton** [comment]
@cdelam The advanced notebook is actually using a hacked version (with residue index offset) of the monomer model to make the predictions for the multimer. Within the new notebook, try set: model_type: alphafold2_ptm

**Comment by SS4620s** [comment]
@sokrypton Thank you very much. This is very helpful. Do you know if there is any documentation about these different model types?
